src/content/docs/en/recipes/i18n.mdx

```diff
@@ -7,9 +7,8 @@ i18nReady: true
 import FileTree from '~/components/FileTree.astro'
 import StaticSsrTabs from '~/components/tabs/StaticSsrTabs.astro'
 
-Astro doesn't have built-in internationalization (i18n) support, but you can build your own i18n solution.
+In v4.0, Astro received [built-in support for internationalization (i18n) routing](/en/guides/internationalization/). However, if you prefer, you can build your own i18n solution. 
```

Comment:
Hey @VoxelMC thanks for handling this, just a tiny suggestion.
```suggestion
In v4.0, Astro received [built-in support for internationalization (i18n) routing](/en/guides/internationalization/). However, if you prefer, you can build your own i18n solution.
```

Reply:
Whoops! Missed that, haha